src/k-provider/ovp/provider.js

```diff
@@ -70,6 +70,27 @@ export default class OVPProvider extends BaseProvider<OVPProviderMediaInfoObject
     });
   }
 
+  doRequest(loaders: Array<ExternalRequestLoader>, params: any = {}): Promise<any> {
+    const dataLoader = new OVPDataLoaderManager(this.playerVersion, this.partnerId, params.ks || this.ks, this._networkRetryConfig);
+
+    return new Promise((resolve, reject) => {
+      loaders.forEach((loaderRequest: ExternalRequestLoader) => {
+        dataLoader.add(loaderRequest.loader, loaderRequest.params);
```

Comment:
loader must include id so add it to the ILoader interface

Reply:
Unfortunately it is not possible to make status properties in an Interface. Will be documented instead